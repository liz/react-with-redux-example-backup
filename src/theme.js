// eslint-disable-next-line import/no-webpack-loader-syntax
import variables from 'sass-extract-loader?{"plugins" :["sass-extract-js"]}!./variables.scss';

export default variables;