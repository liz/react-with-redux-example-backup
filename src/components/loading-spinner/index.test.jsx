import React from 'react';
import { mount } from 'enzyme';

import { LoadingSpinner } from './';
import loading from '../../images/loading.gif';
import theme from '../../theme';

describe('Container', () => {
    it('should match the snapshot', () => {
        const wrapper = mount(<LoadingSpinner />);

        expect(wrapper.html()).toMatchSnapshot();
    });

    it('renders LoadingSpinner with expected props', () => {
        const wrapper = mount(<LoadingSpinner />);

        expect(wrapper.find('LoadingSpinner')).toHaveLength(1);
        expect(wrapper.find('Image').props()).toEqual({
            src: loading,
            alt: "Loading...",
            width: "25px",
            height: "auto",
            horizontalAlignment: "center",
            verticalAlignment: "center",
            maxHeight: "100%",
            maxWidth: "100%",
            type: "tag"
        });
        expect(wrapper.find('LoadingSpinner').props()).toEqual({
            width: "25px"
        });
    });
});