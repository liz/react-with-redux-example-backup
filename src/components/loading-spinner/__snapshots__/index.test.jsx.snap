// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Container should match the snapshot 1`] = `"<div class=\\"loading-spinner__LoadingContainer-vkkyih-0 eooODq\\"><img src=\\"loading.gif\\" width=\\"25px\\" height=\\"auto\\" alt=\\"Loading...\\" class=\\"image__Img-sc-1ai8zv1-2 kXWMGS\\"></div>"`;
